### PR TITLE
Properly display Novaseq timestamp in Genstat

### PR DESCRIPTION
**Pull request by Hammarn**
I also moved the logic out of the the template and into the python script instead.

### CODE REVIEW

File: status/flowcells.py [hunk 1]
```diff
@@ -26,7 +26,14 @@ def list_flowcells(self):
         xfc_view = self.application.x_flowcells_db.view("info/summary",
                                                      descending=True)
         for row in xfc_view:
-            temp_flowcells[row.key] = row.value
+            if len(row.value['startdate']) < 7:
+                row.value['startdate'] = "20{}-{}-{}".format(row.value['startdate'][0:2],row.value['startdate'][2:4],row.value['startdate'][4:6]) 
+                temp_flowcells[row.key] = row.value
```
Comment (ewels) [inline status/flowcells.py]: I think you can just assign to this value directly?

File: status/flowcells.py [hunk 2]
```diff
@@ -26,7 +26,14 @@ def list_flowcells(self):
...
+            else:
+                #Novaseq date format fix
```
Comment (ewels) [inline status/flowcells.py]: Would be nice to paste an example of what this looks like in full

File: status/flowcells.py [hunk 3]
```diff
@@ -26,7 +26,14 @@ def list_flowcells(self):
...
+                mm,dd,yy = row.value['startdate'].split(" ")[0].split("/")
+                row.value['startdate'] = "{}-{}-{}".format(yy,mm,dd)
+                temp_flowcells[row.key] = row.value
```
Comment (ewels) [inline status/flowcells.py]: Nice! Very elegant. Again, I think we can probably assign directly to `row.value['startdate']`

File: status/flowcells.py (hunk 1 above)
Comment (Hammarn) [reply status/flowcells.py]: temp_flowcells[row.key] also needs to be assigned not just row.value['startdate'] being changed. See the code that was there before:
```
temp_flowcells[row.key] = row.value
```

File: status/flowcells.py (hunk 2 above)
Comment (Hammarn) [reply status/flowcells.py]: Good point!

File: status/flowcells.py (hunk 1 above)
Comment (senthil10) [reply status/flowcells.py]: How about using `datetime` as it is already imported ? something like would do

```
row.value['startdate'] = datetime.strptime(row.value['startdate'], "%y%m%d").strftime("%Y-%m-%d")
```

**Comment by Hammarn** [comment]
I talked to @senthil10 an we realized that it's probably better to make sure that the Novaseq date is consistent in the database instead of just when displayed. So I will move my code to FlowcellParser instead:
https://github.com/SciLifeLab/flowcell_parser/blob/master/flowcell_parser/classes.py#L308

File: status/flowcells.py (hunk 1 above)
Comment (ewels) [reply status/flowcells.py]: This is the proper (non-lazy) way to do it 😉

**Comment by ewels** [comment]
Ok great, that's even better 👍

**Comment by Hammarn** [comment]
Ok, so Flowcell_parser code written, tested and merged. I switched to using @senthil10's datetime solution instead so now everything is handled properly 🙂

**Comment by ewels** [comment]
Looks good to me! 👏

**Pull request merged**